kernel/shared/src/main/scala/monocle/Optional.scala

```diff
@@ -15,6 +15,7 @@ trait Optional[A, B] { self =>
   }
 
   def composeLens[C](other: Lens[B, C]): Optional[A, C] = compose(other)
+  def composePrism[C](other: Prism[B, C]): Optional[A, C] = compose(other)
```

Comment:
Are you planning to keep the symbolic operators?

Reply:
I don't know. Maybe not in kernel but we can always have them defined in an extra module. What do you think? In scala 3, we should only have one compose function.